lightkurve/lightcurve.py

```diff
@@ -376,39 +376,42 @@ def flatten(self, window_length=101, polyorder=2, return_trend=False,
             return flatten_lc, trend_lc
         return flatten_lc
 
-    def fold(self, period, transit_midpoint=0.):
-        """Folds the lightcurve at a specified ``period`` and ``transit_midpoint``.
+    def fold(self, period, t0=None, transit_midpoint=0.):
+        """Folds the lightcurve at a specified ``period`` and reference time ``t0``.
 
         This method returns a `FoldedLightCurve` object in which the time
         values range between -0.5 to +0.5 (i.e. the phase).
-        Data points which occur exactly at ``transit_midpoint`` or an integer
-        multiple of ``transit_midpoint + n*period`` will have time value 0.0.
+        Data points which occur exactly at ``t0`` or an integer multiple of
+        ``t0 + n*period`` will have phase value 0.0.
 
         Parameters
         ----------
         period : float
             The period upon which to fold.
+        t0 : float, optional
+            Time corresponding to zero phase. In the same units as the
+            LightCurve's ``time`` attribute.  Defaults to 0.
         transit_midpoint : float, optional
-            Time reference point in the same units as the LightCurve's ``time``
-            attribute.
+            Synonym for `t0`. This parameter will be ignored if ``t0`` is set.
```

Comment:
Why are we letting people set two things? This doesn't seem pythonic...

Reply:
I was thinking that, in this specific case, breaking people's code is worse than being non-Pythonic.  But I suppose I could add a deprecation warning when people use `transit_midpoint`, so we can remove it in a few months!

Reply:
I have gone ahead and added a deprecation warning that will be issued whenever someone uses `transit_midpoint`, and updated the docstring accordingly, so it's more Pythonic know (though won't break anything right away).